microsoft-edge/webview2/get-started/hololens2.md

```diff
@@ -318,87 +260,87 @@ Now that you've reviewed the `WebView` code, let's extend the functionality of o
+               // Set the initial value of the URL input field to the current URL of the WebView
+               if (webView.Page != null)
+               {
+                  URLField.text = webView.Page.AbsoluteUri;
+               }
+         });
+      }
 
-1. In the **Machine Name** field, enter the IP address of your device, as shown below.
-   * You can find the IP address on your HoloLens 2 under **Settings > Network & Internet > Advanced Options**.
-   * We recommend that you manually enter your IP address rather than depend on the "Auto Detected" feature.
+      // Update the URL input field with the new path after navigation
+      private void OnNavigated(string path)
+      {
+         URLField.text = path;
+      }
 
-1. Set the **Authentication Type** to **Universal (Unencrypted Protocol)**:
+      // Enable or disable the Back button based on whether there's any history to go back to
+      private void CanGoBack(bool value)
+      {
+         BackButton.enabled = value;
+      }
+   }
+   ```
 
-   ![The Debugging settings in Project Settings dialog with Machine Name set](./hololens2-images/visualstudio_remote_ip.png)
 
-1. Click the **OK** button.
+1. Connect the UI GameObjects to the WebViewBrowser code that we just wrote:
+   1. Select the WebViewBrowser.
+   1. Drag **Back Button** from the Hierarchy pane into the **Back Button** variable field for the WebView Browser in the Inspector.
+   1. Drag **Go Button** from the Hierarchy pane into the **Go Button** variable field for the WebView Browser in the Inspector:
 
-1. Build, deploy, and debug your app based on your needs:
-   * To build, deploy, and start debugging, select **Debug > Start debugging**.
-   * To build and deploy without debugging, select **Build > Deploy Solution**.
+   ![The WebView Browser Prefab with assigned variables in Unity's Inspector](./hololens2-images/unity_webview_connect_prefabs.png)
 
-1. The first time you deploy an app to your HoloLens 2 from your PC, you'll be prompted for a PIN. Follow the [Pairing your device](/windows/mixed-reality/develop/advanced-concepts/using-visual-studio?tabs=hl2#pairing-your-device) instructions.
+1. You can now test the scene in the Unity Editor. With everything wired up in your scene, you should be able to verify that everything is working as expected. Try testing out the scene by entering a new URL into the `AddressField` text box (must be a full URL including the protocol), then pressing the **Go** button. Make sure the **Back** button works as well.
+    
+    It can sometimes be helpful to debug your application running on your HoloLens, and the steps to do so are slightly different than standard debugging within Visual Studio.   Look at the [Managed debugging with Unity](/windows/mixed-reality/develop/unity/managed-debugging-with-unity-il2cpp) article for more information on how to setup and connect to your application running on the HoloLens or inside of the Unity editor.
```

Comment:
```suggestion
   It can sometimes be helpful to debug your application running on your HoloLens, and the steps to do so are slightly different than standard debugging within Visual Studio.  For more information on how to set up and connect to your application running on the HoloLens or inside of the Unity editor, see [Managed debugging with Unity](/windows/mixed-reality/develop/unity/managed-debugging-with-unity-il2cpp).
```
within a list, must align starting column consistently.
reversed sentence with link, per std pattern